fastlane/spec/actions_specs/commit_version_bump_action_spec.rb

```diff
@@ -8,8 +8,8 @@
     end
 
     it 'returns the param if an Array' do
-      settings_plists = action.settings_plists_from_param ["About.plist", "Root.plist"]
-      expect(settings_plists).to eq ["About.plist", "Root.plist"]
```

Comment:
Just curious why this change is in here?

Reply:
Thanks for the feedback, @ohayon. Maybe I changed this at some point and then reverted it by hand. I can revert this change if desired. I don't have very strong opinions about style. I leave that to RuboCop. Though the `%w` notation is always a little bit shorter.